### PR TITLE
[Bcn WDFT Oct2020] Irene Kyritsis

### PR DESCRIPTION
**Pull request by irenekyri90**
I've added html and css, but am having a bit of a hard time with flexbox. I'm going to go over some details and try to organize CSS a bit better

### CODE REVIEW

**Comment by irenekyri90** [comment]
Made final changes to CSS but couldn't change font-weight.

File: index.html [hunk 1]
```diff
@@ -1,9 +1,102 @@
...
+        <div>
+            <a class="plans-btn" href="#">See plans</a>
+            <a class="free-btn" href="#">Join for free</a>
+        </div>
+    </section>
+
+    <!-------------------Section 2-------------------------->
+
+    <section id="section-2">
+        <img src="images/triangle-hexagon.svg" alt="">
```
Comment (MartaJank) [inline index.html:59]: Remember to always put the description of the image in the "alt" attribute, it is used for blind people to know what is shown in the screen so it's very useful.

File: index.html [hunk 2]
```diff
@@ -1,9 +1,102 @@
...
+            npm is the tool used by over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it. Create an Org and get more out of the tools your team already knows and loves.
+        </article>
+    </section>
+
+    <!-------------------Section 3-------------------------->
+
+    <section id="section-3">
+        <div class="top-container">
+            <div>
+                <img src="images/zero-configuration.svg" alt="zero-configuration">
```
Comment (MartaJank) [inline index.html:71]: Make the alts more descriptive.

File: index.html [hunk 3]
```diff
@@ -1,9 +1,102 @@
...
+                    <li><a href="#">Solutions</a></li>
+                    <li><a href="#">Resources</a></li>
+                    <li><a href="#">Docs</a></li>
+                    <li><a href="#">Support</a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="search-div">
+           <img class="npm-logo" src="images/npm-logo.png" alt="npm-logo">
```
Comment (MartaJank) [inline index.html:30]: Descriptive classes, very good! :)

File: style.css [hunk 4]
```diff
@@ -0,0 +1,211 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
```
Comment (MartaJank) [inline style.css:1]: You can import your font selecting the different weights and italics you want like this (maybe that's why you couldn't deal with font-weight):

You select your styles by clicking on "Select this style":
Untitled

Once you have the ones you like, you can embed them:
Untitled copy

This is another way of embedding the font and a better practice, but you'll learn about that later on (although you can do this as you did but with the different styles selected).
Untitled copy 2

Done! :)

**Pull request closed without merging**